src/main/org/ovirt/sdk/scipt/externalMacsVms.py

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/python
+import getopt
+import sys
+
+from ovirtsdk.api import API
+
+
+def usage():
+    prog_name = sys.argv[0]
+    print('\n%s -l <url> -u <username> -p <password> -c <cert_file>' % prog_name)
+    print('OR')
+    print('%s --url=<url> --username=<username> --password=<password> --certfile=<cert_file path>\n' % prog_name)
+    sys.exit(-1)
+
+
+def process_opts():
+    opts, args = getopt.getopt(args=sys.argv[1:],
+                               shortopts='hl:u:p:c',
+                               longopts=['url=', 'username=', 'password=', 'certfile='])
+
+    url = username = password = cert_file = None
+
+    for opt, arg in opts:
+        if opt == '-h':
+            usage()
+        elif opt in ('-l', '--url'):
+            url = arg
+        elif opt in ('-u', '--username'):
+            username = arg
+        elif opt in ('-p', '--password'):
+            password = arg
+        elif opt in ('-c', '--certfile'):
+            cert_file = arg
+        else:
+            print('Unknown parameter: %s' % opt)
+
+    if url and username and password:
+        return url, username, password, cert_file
+
+    usage()
+
+
+def mac2int(mac):
+    return int(mac.replace(':', ''), 16)
+
+
+def mac_pool2ranges(mac_pool):
```

Comment:
The single caller of the function requires list to be created, so I'd prefer the function to do what it required to rather than being "cool".